packages/tasit-sdk/src/Decentraland.test.js

```diff
@@ -214,6 +233,7 @@ describe("Decentraland", () => {
               `${fingerprint}`
             );
 
+            // eslint-disable-next-line no-unused-vars
             const orderSuccessfulListener = async message => {
```

Comment:
What's the reason this is unused?

Reply:
https://github.com/tasitlabs/TasitSDK/issues/367

Reply:
Ah, got it. Makes sense. Resolving this.